README.md

```diff
@@ -27,6 +27,21 @@ To create a rule that automatically converts Jira issue keys into clickable link
 
 ![An image showing the settings page for LinkMagic with an example pattern](example.png)
 
+## Using RegExp groups
+
+RegExp groups are completely supported, by using the group number or name in the replacement link.
+For example, in a string `There is some target-words-123456 in a sentence` the pattern `target-(?<name>[a-zA-Z]+)-(?<number>[0-9]+)` allows the following replacements:
+
+| replacement | value |
+| ----------- | ----- |
+| `{0}` | "target-words-123456" |
+| `{1}` | "words" |
+| `{2}` | "123456" |
+| `{pattern}` | "target-words-123456" |
+| `{name}` | "words" |
+| `{number}` | "123456" |
+
+A link `https://example.com/{name}/{number}` would then become `https://example.com/words/123456`
```

Comment:
I think this example could be clarified some more.

Maybe a sentence at the end showing what your initial input example sentence (`There is some target-words-123456 in a sentence`) would look like after the plugin runs.

Reply:
Updated in 4577458